README.md

```diff
@@ -282,7 +282,8 @@ And finally, you are ready to go! Or at least you should be...
 **Iris Mestres** | [Contact me][deliaBlue]
 
 ### Junior Contributors
-
+**Stephanie Padilla** | [MStephPad] [MStephPad] 
+[MStephPad]: <https://github.com/MstephPad>
```

Comment:
Almost Perfect!

You must put the link with all the other links. My recomendation it is to place it on the top part,  in the bottom or if you want to do it in the best way sorted alphabetically.